Import SAF via URL no longer works

**Describe the bug**
Actually the import of a SAF via remote URL is not working.

**To Reproduce**
Steps to reproduce the behavior:
1. Login as an Admin
2. Go to "Processes"
3. Create a new process "import"
4. Add the `--add` parameter
5. Add the `--url` parameter: https://github.com/DSpace-Labs/AIP-Files/releases/download/demo-entities-data/SAF-Import-Entities.zip
6. Add the `--collection` parameter specifying the UUID of a collection
7. Click "Save"

After a few seconds, this error appears:
````
2023-06-06 21:51:58.549 INFO import - 54 @ The script has started
2023-06-06 21:51:58.551 INFO import - 54 @ Destination collections:
2023-06-06 21:51:58.551 INFO import - 54 @ Owning Collection: Test Import from URL Collection
2023-06-06 21:51:59.075 INFO import - 54 @ Started: 1686088318549
2023-06-06 21:51:59.075 INFO import - 54 @ Ended: 1686088319074
2023-06-06 21:51:59.075 INFO import - 54 @ Elapsed time: 0 secs (525 msecs)
2023-06-06 21:51:59.075 ERROR import - 54 @ null
2023-06-06 21:51:59.075 ERROR import - 54 @ java.lang.Exception: Error committing changes to database: Authorization attempted on null DSpace object READ by user 335647b6-8a52-4ecb-a8c1-7ebabb199bda, aborting most recent changes
	at org.dspace.app.itemimport.ItemImport.internalRun(ItemImport.java:233)
	at org.dspace.scripts.DSpaceRunnable.run(DSpaceRunnable.java:104)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:829)
Caused by: org.dspace.authorize.AuthorizeException: Authorization attempted on null DSpace object READ by user 335647b6-8a52-4ecb-a8c1-7ebabb199bda
	at org.dspace.authorize.AuthorizeServiceImpl.authorizeAction(AuthorizeServiceImpl.java:144)
	at org.dspace.authorize.AuthorizeServiceImpl.authorizeAction(AuthorizeServiceImpl.java:120)
	at org.dspace.authorize.AuthorizeServiceImpl.authorizeAction(AuthorizeServiceImpl.java:114)
	at org.dspace.content.BitstreamServiceImpl.retrieve(BitstreamServiceImpl.java:298)
	at org.dspace.app.rest.scripts.handler.impl.RestDSpaceRunnableHandler.getFileStream(RestDSpaceRunnableHandler.java:237)
	at org.dspace.app.itemimport.ItemImport.readZip(ItemImport.java:345)
	at org.dspace.app.itemimport.ItemImport.internalRun(ItemImport.java:224)
	... 4 more
````

**Expected behavior**
The import SAF via remote URL should work.

**Related work**
This problem is blocking this issue https://github.com/DSpace/dspace-angular/issues/1913 (with related pull request https://github.com/DSpace/dspace-angular/pull/2212)

--- a/dspace-api/src/main/java/org/dspace/app/itemimport/ItemImport.java
+++ b/dspace-api/src/main/java/org/dspace/app/itemimport/ItemImport.java
@@ -332,17 +332,19 @@ public class ItemImport extends DSpaceRunnable<ItemImportScriptConfiguration> {
      */
     protected void readZip(Context context, ItemImportService itemImportService) throws Exception {
         Optional<InputStream> optionalFileStream = Optional.empty();
+        Optional<InputStream> validationFileStream = Optional.empty();
         if (!remoteUrl) {
             // manage zip via upload
             optionalFileStream = handler.getFileStream(context, zipfilename);
+            validationFileStream = handler.getFileStream(context, zipfilename);
         } else {
             // manage zip via remote url
             optionalFileStream = Optional.ofNullable(new URL(zipfilename).openStream());
+            validationFileStream = Optional.ofNullable(new URL(zipfilename).openStream());
         }
 
-        if (optionalFileStream.isPresent()) {
+        if (validationFileStream.isPresent()) {
             // validate zip file
-            Optional<InputStream> validationFileStream = handler.getFileStream(context, zipfilename);
             if (validationFileStream.isPresent()) {
                 validateZip(validationFileStream.get());
             }

